java/mvnw

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
```

Comment:
This file is a simplified version of Apache Spark mvn wrapper.
- https://github.com/apache/spark/blob/master/build/mvn